src/containers/UserProfile/LeftPanel.js

```diff
@@ -1,30 +1,40 @@
 import React, { Component, Fragment } from "react";
-import EditIntro from "../../containers/UserProfile/EditIntro";
+import EditIntro from "./EditIntro";
 import ReactNotification from "react-notifications-component";
 import "react-notifications-component/dist/theme.css";
 import { addNotification, getInitials } from "../../utilities/index";
 import * as auth from "../../services/Session";
-import "./LeftPane.scss";
+import "./LeftPanel.scss";
 import AboutUser from "./AboutUser";
 
 class LeftPane extends Component {
   constructor(props) {
     super(props);
-    this.state = { readOnly: true, open: true };
-    this.closeModal = this.closeModal.bind(this);
+    this.state = {
+      readOnly: true,
+      open: true
+    };
     this.notificationDOMRef = React.createRef();
     this.formRef = null;
   }
 
-  closeModal() {
+  closeModal = () => {
     this.setState({ open: false });
     addNotification(
       this.notificationDOMRef,
       "success",
       "success",
       "User has been updated"
     );
-  }
+  };
+
+  getAboutValue = about => {
```

Comment:
@Humnaa use ternary operator here instead. It will be short and more convenient.

Reply:
I have replaced it with ternary operator